### ISSUE TITLE
4.14.0-rc1: include alloc.h bit.h not found

### ISSUE THREAD

**Issue by jubalh** [shadow-maint/shadow#763]
```
[    2s] + export 'CFLAGS=-O2 -Wall -U_FORTIFY_SOURCE -D_FORTIFY_SOURCE=3 -fstack-protector-strong -funwind-tables -fasynchronous-unwind-tables -fstack-clash-protection -Werror=return-type -flto=auto -g -fpie'
[    2s] + CFLAGS='-O2 -Wall -U_FORTIFY_SOURCE -D_FORTIFY_SOURCE=3 -fstack-protector-strong -funwind-tables -fasynchronous-unwind-tables -fstack-clash-protection -Werror=return-type -flto=auto -g -fpie'
[    2s] + export LDFLAGS=-pie
[    2s] + LDFLAGS=-pie
[    2s] + autoreconf -fvi
[    2s] autoreconf: export WARNINGS=
[    2s] autoreconf: Entering directory '.'
[    2s] autoreconf: running: autopoint --force
[    3s] Copying file m4/codeset.m4
[    3s] Copying file m4/extern-inline.m4
[    3s] Copying file m4/fcntl-o.m4
[    3s] Copying file m4/glibc2.m4
[    3s] Copying file m4/glibc21.m4
[    3s] Copying file m4/intdiv0.m4
[    3s] Copying file m4/intl.m4
[    3s] Copying file m4/intldir.m4
[    3s] Copying file m4/intmax.m4
[    3s] Copying file m4/inttypes-pri.m4
[    3s] Copying file m4/inttypes_h.m4
[    3s] Copying file m4/lcmessage.m4
[    3s] Copying file m4/lock.m4
[    3s] Copying file m4/longlong.m4
[    3s] Copying file m4/printf-posix.m4
[    3s] Copying file m4/size_max.m4
[    3s] Copying file m4/stdint_h.m4
[    3s] Copying file m4/threadlib.m4
[    3s] Copying file m4/uintmax_t.m4
[    3s] Copying file m4/visibility.m4
[    3s] Copying file m4/wchar_t.m4
[    3s] Copying file m4/wint_t.m4
[    3s] Copying file m4/xsize.m4
[    3s] Copying file po/Makevars.template
[    5s] autoreconf: running: aclocal --force 
[    5s] autoreconf: configure.ac: tracing
[    6s] autoreconf: running: libtoolize --copy --force
[    6s] libtoolize: putting auxiliary files in '.'.
[    6s] libtoolize: copying file './ltmain.sh'
[    6s] libtoolize: putting macros in AC_CONFIG_MACRO_DIRS, 'm4'.
[    6s] libtoolize: copying file 'm4/libtool.m4'
[    6s] libtoolize: copying file 'm4/ltoptions.m4'
[    6s] libtoolize: copying file 'm4/ltsugar.m4'
[    6s] libtoolize: copying file 'm4/ltversion.m4'
[    6s] libtoolize: copying file 'm4/lt~obsolete.m4'
[    6s] libtoolize: Consider adding '-I m4' to ACLOCAL_AMFLAGS in Makefile.am.
[    6s] autoreconf: configure.ac: not using Intltool
[    6s] autoreconf: configure.ac: not using Gtkdoc
[    6s] autoreconf: running: aclocal --force 
[    7s] autoreconf: running: /usr/bin/autoconf --force
[    7s] autoreconf: running: /usr/bin/autoheader --force
[    7s] autoreconf: running: automake --add-missing --copy --force-missing
[    8s] configure.ac:23: installing './compile'
[    8s] configure.ac:9: installing './missing'
[    8s] lib/Makefile.am: installing './depcomp'
[    8s] autoreconf: Leaving directory '.'
[    8s] + CONFIG_SHELL=/usr/bin/bash
[    8s] + export CONFIG_SHELL
[    8s] + CFLAGS='-O2 -Wall -U_FORTIFY_SOURCE -D_FORTIFY_SOURCE=3 -fstack-protector-strong -funwind-tables -fasynchronous-unwind-tables -fstack-clash-protection -Werror=return-type -flto=auto -g -fpie'
[    8s] + export CFLAGS
[    8s] + CXXFLAGS='-O2 -Wall -U_FORTIFY_SOURCE -D_FORTIFY_SOURCE=3 -fstack-protector-strong -funwind-tables -fasynchronous-unwind-tables -fstack-clash-protection -Werror=return-type -flto=auto -g'
[    8s] + export CXXFLAGS
[    8s] + FFLAGS='-O2 -Wall -U_FORTIFY_SOURCE -D_FORTIFY_SOURCE=3 -fstack-protector-strong -funwind-tables -fasynchronous-unwind-tables -fstack-clash-protection -Werror=return-type -flto=auto -g '
[    8s] + export FFLAGS
[    8s] + FCFLAGS='-O2 -Wall -U_FORTIFY_SOURCE -D_FORTIFY_SOURCE=3 -fstack-protector-strong -funwind-tables -fasynchronous-unwind-tables -fstack-clash-protection -Werror=return-type -flto=auto -g '
[    8s] + export FCFLAGS
[    8s] + LDFLAGS=-pie
[    8s] + export LDFLAGS
[    8s] + ./configure --host=x86_64-suse-linux-gnu --build=x86_64-suse-linux-gnu --program-prefix= --disable-dependency-tracking --prefix=/usr --exec-prefix=/usr --bindir=/usr/bin --sbindir=/usr/sbin --sysconfdir=/etc --datadir=/usr/share --includedir=/usr/include --libdir=/usr/lib64 --libexecdir=/usr/libexec --localstatedir=/var --sharedstatedir=/var/lib --mandir=/usr/share/man --infodir=/usr/share/info --enable-shadowgrp --enable-account-tools-setuid --with-audit --with-libpam --with-sha-crypt --with-acl --with-attr --with-nscd --with-selinux --without-libcrack --with-group-name-max-length=32 --enable-vendordir=/usr/etc
[    8s] checking for a BSD-compatible install... /usr/bin/install -c
[    8s] checking whether build environment is sane... yes
[    8s] checking for a race-free mkdir -p... /usr/bin/mkdir -p
[    8s] checking for gawk... gawk
[    8s] checking whether make sets $(MAKE)... yes
[    8s] checking whether make supports nested variables... yes
[    8s] checking whether make supports nested variables... (cached) yes
[    8s] checking whether make supports the include directive... yes (GNU style)
[    8s] checking for x86_64-suse-linux-gnu-gcc... no
[    8s] checking for gcc... gcc
[    8s] checking whether the C compiler works... yes
[    8s] checking for C compiler default output file name... a.out
[    9s] checking for suffix of executables... 
[    9s] checking whether we are cross compiling... no
[    9s] checking for suffix of object files... o
[    9s] checking whether the compiler supports GNU C... yes
[    9s] checking whether gcc accepts -g... yes
[    9s] checking for gcc option to enable C11 features... none needed
[    9s] checking whether gcc understands -c and -o together... yes
[    9s] checking dependency style of gcc... none
[    9s] checking for stdio.h... yes
[    9s] checking for stdlib.h... yes
[    9s] checking for string.h... yes
[    9s] checking for inttypes.h... yes
[    9s] checking for stdint.h... yes
[    9s] checking for strings.h... yes
[    9s] checking for sys/stat.h... yes
[    9s] checking for sys/types.h... yes
[    9s] checking for unistd.h... yes
[    9s] checking for wchar.h... yes
[    9s] checking for minix/config.h... no
[    9s] checking for utime.h... yes
[    9s] checking whether it is safe to define __EXTENSIONS__... yes
[    9s] checking whether _XOPEN_SOURCE should be defined... no
[    9s] checking whether to enable maintainer-specific portions of Makefiles... no
[    9s] checking for x86_64-suse-linux-gnu-gcc... gcc
[    9s] checking whether the compiler supports GNU C... (cached) yes
[    9s] checking whether gcc accepts -g... (cached) yes
[    9s] checking for gcc option to enable C11 features... (cached) none needed
[    9s] checking whether gcc understands -c and -o together... (cached) yes
[    9s] checking dependency style of gcc... (cached) none
[    9s] checking whether ln -s works... yes
[    9s] checking for bison... no
[    9s] checking for byacc... no
[    9s] checking build system type... x86_64-suse-linux-gnu
[    9s] checking host system type... x86_64-suse-linux-gnu
[    9s] checking how to print strings... printf
[    9s] checking for a sed that does not truncate output... /usr/bin/sed
[    9s] checking for grep that handles long lines and -e... /usr/bin/grep
[    9s] checking for egrep... /usr/bin/grep -E
[    9s] checking for fgrep... /usr/bin/grep -F
[    9s] checking for ld used by gcc... /usr/x86_64-suse-linux/bin/ld
[    9s] checking if the linker (/usr/x86_64-suse-linux/bin/ld) is GNU ld... yes
[    9s] checking for BSD- or MS-compatible name lister (nm)... /usr/bin/nm -B
[    9s] checking the name lister (/usr/bin/nm -B) interface... BSD nm
[    9s] checking the maximum length of command line arguments... 1572864
[    9s] checking how to convert x86_64-suse-linux-gnu file names to x86_64-suse-linux-gnu format... func_convert_file_noop
[    9s] checking how to convert x86_64-suse-linux-gnu file names to toolchain format... func_convert_file_noop
[    9s] checking for /usr/x86_64-suse-linux/bin/ld option to reload object files... -r
[    9s] checking for x86_64-suse-linux-gnu-file... no
[    9s] checking for file... file
[    9s] checking for x86_64-suse-linux-gnu-objdump... no
[    9s] checking for objdump... objdump
[    9s] checking how to recognize dependent libraries... pass_all
[    9s] checking for x86_64-suse-linux-gnu-dlltool... no
[    9s] checking for dlltool... no
[    9s] checking how to associate runtime and link libraries... printf %s\n
[    9s] checking for x86_64-suse-linux-gnu-ar... no
[    9s] checking for ar... ar
[    9s] checking for archiver @FILE support... @
[    9s] checking for x86_64-suse-linux-gnu-strip... no
[    9s] checking for strip... strip
[    9s] checking for x86_64-suse-linux-gnu-ranlib... no
[    9s] checking for ranlib... ranlib
[    9s] checking command to parse /usr/bin/nm -B output from gcc object... ok
[    9s] checking for sysroot... no
[    9s] checking for a working dd... /usr/bin/dd
[    9s] checking how to truncate binary pipes... /usr/bin/dd bs=4096 count=1
[    9s] checking for x86_64-suse-linux-gnu-mt... no
[    9s] checking for mt... no
[    9s] checking if : is a manifest tool... no
[   10s] checking for dlfcn.h... yes
[   10s] checking for objdir... .libs
[   10s] checking if gcc supports -fno-rtti -fno-exceptions... no
[   10s] checking for gcc option to produce PIC... -fPIC -DPIC
[   10s] checking if gcc PIC flag -fPIC -DPIC works... yes
[   10s] checking if gcc static flag -static works... no
[   10s] checking if gcc supports -c -o file.o... yes
[   10s] checking if gcc supports -c -o file.o... (cached) yes
[   10s] checking whether the gcc linker (/usr/x86_64-suse-linux/bin/ld -m elf_x86_64) supports shared libraries... yes
[   10s] checking whether -lc should be explicitly linked in... no
[   10s] checking dynamic linker characteristics... GNU/Linux ld.so
[   10s] checking how to hardcode library paths into programs... immediate
[   10s] checking whether stripping libraries is possible... yes
[   10s] checking if libtool supports shared libraries... yes
[   10s] checking whether to build shared libraries... yes
[   10s] checking whether to build static libraries... yes
[   10s] checking for crypt.h... yes
[   10s] checking for utmp.h... yes
[   10s] checking for termio.h... yes
[   10s] checking for sgtty.h... yes
[   10s] checking for sys/ioctl.h... yes
[   10s] checking for paths.h... yes
[   10s] checking for sys/capability.h... no
[   10s] checking for sys/random.h... yes
[   10s] checking for gshadow.h... yes
[   10s] checking for lastlog.h... yes
[   10s] checking for rpc/key_prot.h... no
[   10s] checking for acl/libacl.h... yes
[   10s] checking for attr/libattr.h... yes
[   10s] checking for attr/error_context.h... yes
[   10s] checking for shadow.h... yes
[   10s] checking for arc4random_buf... yes
[   10s] checking for futimes... yes
[   11s] checking for getentropy... yes
[   11s] checking for getrandom... yes
[   11s] checking for getspnam... yes
[   11s] checking for getusershell... yes
[   11s] checking for initgroups... yes
[   11s] checking for lckpwdf... yes
[   11s] checking for lutimes... yes
[   11s] checking for mempcpy... yes
[   11s] checking for setgroups... yes
[   11s] checking for updwtmp... yes
[   11s] checking for updwtmpx... yes
[   12s] checking for innetgr... yes
[   12s] checking for getspnam_r... yes
[   12s] checking for rpmatch... yes
[   12s] checking for memset_explicit... no
[   12s] checking for explicit_bzero... yes
[   12s] checking for stpecpy... no
[   12s] checking for stpeprintf... no
[   12s] checking for special C compiler options needed for large files... no
[   12s] checking for _FILE_OFFSET_BITS value needed for large files... no
[   12s] checking for struct utmp.ut_type... yes
[   12s] checking for struct utmp.ut_id... yes
[   12s] checking for struct utmp.ut_name... yes
[   12s] checking for struct utmp.ut_user... yes
[   12s] checking for struct utmp.ut_host... yes
[   12s] checking for struct utmp.ut_syslen... no
[   12s] checking for struct utmp.ut_addr... yes
[   12s] checking for struct utmp.ut_addr_v6... yes
[   12s] checking for struct utmp.ut_time... yes
[   12s] checking for struct utmp.ut_xtime... yes
[   12s] checking for struct utmp.ut_tv... yes
[   12s] checking how to run the C preprocessor... gcc -E
[   12s] checking for uid_t in sys/types.h... yes
[   12s] checking type of array argument to getgroups... gid_t
[   13s] checking whether utime accepts a null argument... yes
[   13s] checking for putgrent... yes
[   13s] checking for putpwent... yes
[   13s] checking for putspent... yes
[   13s] checking for sgetgrent... no
[   13s] checking for sgetpwent... no
[   13s] checking for sgetspent... yes
[   13s] checking for setpgrp... yes
[   13s] checking for secure_getenv... yes
[   13s] checking for working shadow group support... yes
[   13s] checking location of shared mail directory... /var/mail
[   13s] checking location of user mail file... none
[   13s] checking location of utmp... none
[   13s] configure: WARNING: utmp file not found
[   13s] checking location of faillog/lastlog/wtmp... /var/log
[   13s] checking location of the passwd program... /usr/bin
[   13s] checking for posix_spawn... yes
[   13s] checking for posix_spawn... (cached) yes
[   13s] checking for library containing gethostbyname... none required
[   13s] checking for econf_readDirs in -leconf... yes
[   13s] checking size of uid_t... 4
[   14s] checking size of gid_t... 4
[   14s] checking for sd_session_get_remote_host in -lsystemd... no
[   14s] checking for crypt in -lcrypt... yes
[   14s] checking for crypt in -lcrypt... (cached) yes
[   14s] checking for library containing readpassphrase... -lbsd
[   14s] checking for x86_64-suse-linux-gnu-pkg-config... /usr/bin/x86_64-suse-linux-gnu-pkg-config
[   14s] checking pkg-config is at least version 0.9.0... yes
[   14s] checking for LIBBSD... yes
[   14s] checking for readpassphrase.h... yes
[   14s] checking for acl/libacl.h... (cached) yes
[   14s] checking for attr/error_context.h... (cached) yes
[   14s] checking for perm_copy_file in -lacl... yes
[   14s] checking for perm_copy_fd in -lacl... yes
[   14s] checking for attr/libattr.h... (cached) yes
[   14s] checking for attr/error_context.h... (cached) yes
[   14s] checking for attr_copy_file in -lattr... yes
[   14s] checking for attr_copy_fd in -lattr... yes
[   14s] checking for libaudit.h... yes
[   14s] checking for gcc options needed to detect all undeclared functions... none needed
[   14s] checking whether AUDIT_ADD_USER is declared... yes
[   14s] checking whether AUDIT_DEL_USER is declared... yes
[   14s] checking whether AUDIT_ADD_GROUP is declared... yes
[   14s] checking whether AUDIT_DEL_GROUP is declared... yes
[   14s] checking for audit_log_acct_message in -laudit... yes
[   14s] checking for sys/statfs.h... yes
[   14s] checking for linux/magic.h... yes
[   14s] checking for linux/btrfs_tree.h... yes
[   14s] checking for selinux/selinux.h... yes
[   14s] checking for semanage/semanage.h... yes
[   14s] checking for is_selinux_enabled in -lselinux... yes
[   14s] checking for semanage_connect in -lsemanage... yes
[   14s] checking for tcb.h... no
[   14s] checking for pam_start in -lpam... yes
[   15s] checking for openpam_ttyconv in -lpam... no
[   15s] checking for misc_conv in -lpam_misc... yes
[   15s] checking for security/openpam.h... no
[   15s] checking for security/pam_misc.h... yes
[   15s] checking whether PAM_ESTABLISH_CRED is declared... yes
[   15s] checking whether PAM_DELETE_CRED is declared... yes
[   15s] checking whether PAM_NEW_AUTHTOK_REQD is declared... yes
[   15s] checking whether PAM_DATA_SILENT is declared... yes
[   15s] checking for pam_fail_delay... yes
[   15s] checking use login and su access checking if PAM not used... no
[   15s] checking for fgetpwent_r... yes
[   15s] checking whether NLS is requested... yes
[   15s] checking for msgfmt... /usr/bin/msgfmt
[   15s] checking for gmsgfmt... /usr/bin/msgfmt
[   15s] checking for xgettext... /usr/bin/xgettext
[   15s] checking for msgmerge... /usr/bin/msgmerge
[   15s] checking for ld used by gcc... /usr/x86_64-suse-linux/bin/ld -m elf_x86_64
[   15s] checking if the linker (/usr/x86_64-suse-linux/bin/ld -m elf_x86_64) is GNU ld... yes
[   15s] checking for shared library run path origin... done
[   15s] checking for CFPreferencesCopyAppValue... no
[   15s] checking for CFLocaleCopyCurrent... no
[   15s] checking for GNU gettext in libc... yes
[   15s] checking whether to use NLS... yes
[   15s] checking where the gettext function comes from... libc
[   15s] checking that generated files are newer than configure... done
[   15s] configure: creating ./config.status
[   15s] config.status: creating Makefile
[   15s] config.status: creating po/Makefile.in
[   15s] config.status: creating doc/Makefile
[   15s] config.status: creating man/Makefile
[   15s] config.status: creating man/config.xml
[   15s] config.status: creating man/po/Makefile
[   15s] config.status: creating man/cs/Makefile
[   15s] config.status: creating man/da/Makefile
[   15s] config.status: creating man/de/Makefile
[   15s] config.status: creating man/es/Makefile
[   15s] config.status: creating man/fi/Makefile
[   15s] config.status: creating man/fr/Makefile
[   15s] config.status: creating man/hu/Makefile
[   15s] config.status: creating man/id/Makefile
[   15s] config.status: creating man/it/Makefile
[   15s] config.status: creating man/ja/Makefile
[   15s] config.status: creating man/ko/Makefile
[   15s] config.status: creating man/pl/Makefile
[   15s] config.status: creating man/pt_BR/Makefile
[   15s] config.status: creating man/ru/Makefile
[   15s] config.status: creating man/sv/Makefile
[   15s] config.status: creating man/tr/Makefile
[   15s] config.status: creating man/uk/Makefile
[   15s] config.status: creating man/zh_CN/Makefile
[   15s] config.status: creating man/zh_TW/Makefile
[   15s] config.status: creating libmisc/Makefile
[   16s] config.status: creating lib/Makefile
[   16s] config.status: creating libsubid/Makefile
[   16s] config.status: creating libsubid/subid.h
[   16s] config.status: creating src/Makefile
[   16s] config.status: creating contrib/Makefile
[   16s] config.status: creating etc/Makefile
[   16s] config.status: creating etc/pam.d/Makefile
[   16s] config.status: creating shadow.spec
[   16s] config.status: creating config.h
[   16s] config.status: executing depfiles commands
[   16s] config.status: executing libtool commands
[   16s] config.status: executing po-directories commands
[   16s] config.status: creating po/POTFILES
[   16s] config.status: creating po/Makefile
[   16s] shadow will be compiled with the following features:
[   16s] 
[   16s] 	auditing support:		yes
[   16s] 	CrackLib support:		no
[   16s] 	PAM support:			yes
[   16s] 	suid account management tools:	yes
[   16s] 	SELinux support:		yes
[   16s] 	BtrFS support:			yes
[   16s] 	ACL support:			yes
[   16s] 	Extended Attributes support:	yes
[   16s] 	tcb support (incomplete):	no
[   16s] 	shadow group support:		yes
[   16s] 	S/Key support:			no
[   16s] 	SHA passwords encryption:	yes
[   16s] 	bcrypt passwords encryption:	no
[   16s] 	yescrypt passwords encryption:	no
[   16s] 	nscd support:			yes
[   16s] 	sssd support:			yes
[   16s] 	subordinate IDs support:	yes
[   16s] 	enable lastlog:			no
[   16s] 	enable logind:			no
[   16s] 	use file caps:			no
[   16s] 	install su:			yes
[   16s] 	enabled vendor dir:             /usr/etc
[   16s] 
[   16s] + /usr/bin/make -O -j8 V=1 VERBOSE=1
[   16s] /usr/bin/make  all-recursive
[   16s] Making all in libmisc
[   16s] make[2]: Entering directory '/home/abuild/rpmbuild/BUILD/shadow-4.14.0-rc1/libmisc'
[   16s] /usr/bin/bash ../libtool  --tag=CC   --mode=compile gcc -DHAVE_CONFIG_H -I. -I..  -I../lib -I.. -DUSE_ECONF=1  -isystem /usr/include/bsd -DLIBBSD_OVERLAY  -lcrypt  -O2 -Wall -U_FORTIFY_SOURCE -D_FORTIFY_SOURCE=3 -fstack-protector-strong -funwind-tables -fasynchronous-unwind-tables -fstack-clash-protection -Werror=return-type -flto=auto -g -fpie -c -o libmisc_la-alloc.lo `test -f 'alloc.c' || echo './'`alloc.c
[   16s] libtool: compile:  gcc -DHAVE_CONFIG_H -I. -I.. -I../lib -I.. -DUSE_ECONF=1 -isystem /usr/include/bsd -DLIBBSD_OVERLAY -lcrypt -O2 -Wall -U_FORTIFY_SOURCE -D_FORTIFY_SOURCE=3 -fstack-protector-strong -funwind-tables -fasynchronous-unwind-tables -fstack-clash-protection -Werror=return-type -flto=auto -g -c alloc.c  -fPIC -DPIC -o .libs/libmisc_la-alloc.o
[   16s] alloc.c:24:10: fatal error: alloc.h: No such file or directory
[   16s]    24 | #include "alloc.h"
[   16s]       |          ^~~~~~~~~
[   16s] compilation terminated.
[   16s] make[2]: *** [Makefile:693: libmisc_la-alloc.lo] Error 1
[   16s] make[2]: Leaving directory '/home/abuild/rpmbuild/BUILD/shadow-4.14.0-rc1/libmisc'
[   16s] make[2]: *** Waiting for unfinished jobs....
[   16s] make[2]: Entering directory '/home/abuild/rpmbuild/BUILD/shadow-4.14.0-rc1/libmisc'
[   16s] /usr/bin/bash ../libtool  --tag=CC   --mode=compile gcc -DHAVE_CONFIG_H -I. -I..  -I../lib -I.. -DUSE_ECONF=1  -isystem /usr/include/bsd -DLIBBSD_OVERLAY  -lcrypt  -O2 -Wall -U_FORTIFY_SOURCE -D_FORTIFY_SOURCE=3 -fstack-protector-strong -funwind-tables -fasynchronous-unwind-tables -fstack-clash-protection -Werror=return-type -flto=auto -g -fpie -c -o libmisc_la-bit.lo `test -f 'bit.c' || echo './'`bit.c
[   16s] libtool: compile:  gcc -DHAVE_CONFIG_H -I. -I.. -I../lib -I.. -DUSE_ECONF=1 -isystem /usr/include/bsd -DLIBBSD_OVERLAY -lcrypt -O2 -Wall -U_FORTIFY_SOURCE -D_FORTIFY_SOURCE=3 -fstack-protector-strong -funwind-tables -fasynchronous-unwind-tables -fstack-clash-protection -Werror=return-type -flto=auto -g -c bit.c  -fPIC -DPIC -o .libs/libmisc_la-bit.o
[   16s] bit.c:12:10: fatal error: bit.h: No such file or directory
[   16s]    12 | #include "bit.h"
[   16s]       |          ^~~~~~~
[   16s] compilation terminated.
[   16s] make[2]: *** [Makefile:714: libmisc_la-bit.lo] Error 1
[   16s] make[2]: Leaving directory '/home/abuild/rpmbuild/BUILD/shadow-4.14.0-rc1/libmisc'
[   16s] make[2]: Entering directory '/home/abuild/rpmbuild/BUILD/shadow-4.14.0-rc1/libmisc'
[   16s] /usr/bin/bash ../libtool  --tag=CC   --mode=compile gcc -DHAVE_CONFIG_H -I. -I..  -I../lib -I.. -DUSE_ECONF=1  -isystem /usr/include/bsd -DLIBBSD_OVERLAY  -lcrypt  -O2 -Wall -U_FORTIFY_SOURCE -D_FORTIFY_SOURCE=3 -fstack-protector-strong -funwind-tables -fasynchronous-unwind-tables -fstack-clash-protection -Werror=return-type -flto=auto -g -fpie -c -o libmisc_la-agetpass.lo `test -f 'agetpass.c' || echo './'`agetpass.c
[   16s] libtool: compile:  gcc -DHAVE_CONFIG_H -I. -I.. -I../lib -I.. -DUSE_ECONF=1 -isystem /usr/include/bsd -DLIBBSD_OVERLAY -lcrypt -O2 -Wall -U_FORTIFY_SOURCE -D_FORTIFY_SOURCE=3 -fstack-protector-strong -funwind-tables -fasynchronous-unwind-tables -fstack-clash-protection -Werror=return-type -flto=auto -g -c agetpass.c  -fPIC -DPIC -o .libs/libmisc_la-agetpass.o
[   16s] agetpass.c:18:10: fatal error: alloc.h: No such file or directory
[   16s]    18 | #include "alloc.h"
[   16s]       |          ^~~~~~~~~
[   16s] compilation terminated.
```

**Comment by jubalh** [comment]
`alloc.h` and `bit.h` are in `lib`.
`alloc.c` and `bit.c` are in `libmisc`?

**Comment by hallyn** [comment]
That sounds right.  Fixes: 4a56f2baabd I guess.

**Comment by jubalh** [comment]
@hallyn 
BTW in https://github.com/shadow-maint/shadow/releases/download/4.14.0-rc1/shadow-4.14.0-rc1.tar.gz (manually uploaded) I don't find the alloc.h and bit.h file.
In https://github.com/shadow-maint/shadow/archive/refs/tags/4.14.0-rc1.tar.gz they are present.
